docs/react-v9/contributing/rfcs/react-components/convergence/extended-design-tokens.md

```diff
@@ -0,0 +1,442 @@
+| C: In a new package, e.g. @fluentui/tokens-expanded                                                  | • Doesn't introduce circular dependency<br>• Flexibility around packaging & versioning<br>• Insulates @fluentui/tokens from control token churn | • More friction to opt-in to expanded tokens<br>• Adds overhead                                                                                                                                               |
+
+## How far do we go in exposing semantic tokens?
+
+The expanded token system opens the door to adding many more tokens than might be needed while growing the API surface. We should consider the strategy we’ll use to scope and prioritize which components and variants will be tokenized, and how tokens will be exposed over time. We’re leaning towards Option C (**bolded**) but would like feedback.
+
+| Option                                                                                                                                                                                                                                                    | Pros                                                                                                                                                                | Cons                                                                                                                                                                                                      |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Option A: Tokenize everything, even if it's a variant. This includes recreating tokens for those variants even if the base has tokens.<br><br>e.g. base button includes ctrlButtonBackgroundRest, rounded would then have ctrlButtonRoundedBackgroundRest | • Tokens can control all aspects of each control and its variants.                                                                                                  | • Huge API surface<br>• A lot more work to implement<br>• Could easily get out of hand<br>• Hard to roll back (we basically couldn't) if we find we don't need this level of fidelity                     |
```

Comment:
Is there an analysis of how many tokens we might add for each of these options? For example, look at `Button` and `Combobox`, and see how many net new tokens we end up and use that to extrapolate how many we might add for the entire component library?

Reply:
I think we're looking at about 411 Semantic tokens and 1110 control tokens at the moment. Keeping in mind none of these have values and will fall back to existing Alias (Fluent 2) tokens.

In terms of `Button` it would be on the order of 50+ semantic tokens (shared with other components) and 100+ control tokens.

Reply:
Update is that we're looking at 370 Semantic & 350 Control tokens.